conversation/src/main/java/com/ibm/watson/developer_cloud/conversation/v1/model/UpdateWorkspace.java

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.conversation.v1.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * UpdateWorkspace.
+ */
+public class UpdateWorkspace extends GenericModel {
```

Comment:
We could do that but then it should be changed in the swagger.  That is how it is handled in NLU.  `AnalyzeOptions`, `CategoriesOptions`, `EmotionOptions`, `EntitiesOptions` etc all are named this way in the NLU swagger doc.